packages/react-native-client/src/components/VideoRendererView.tsx

```diff
@@ -14,6 +14,7 @@ export type VideoRendererProps = {
    * @default `FILL`
    */
   videoLayout?: VideoLayout;
+  alwaysRender?: boolean;
```

Comment:
**suggestion:** Can you please document and describe what this property does, when and why should user set it.